pkg/distributor/validator.go

```diff
@@ -143,7 +143,11 @@ func (v Validator) ValidateEntry(ctx context.Context, vCtx validationContext, la
 	return nil
 }
 
-// Validate labels returns an error if the labels are invalid
+func (v Validator) isAggregatedMetricStream(ls labels.Labels) bool {
```

Comment:
should this be public at this point? I can see either way since this is all in the distributor package, but it feels public to me now :)

Reply:
the aggregated metric itself is public already, so +1 on making this public.